zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -70,31 +73,36 @@ trait ZipkinWebFactory { self: App =>
     stats: StatsReceiver = DefaultStatsReceiver.scope("zipkin-web")
   ): Service[Request, Response] = {
     val handlers = newHandlers
-    import handlers._
+    import handlers.{handleIndexHtml, handleConfig, handlePublic, handleRoute, handleTrace, collectStats, renderPage, catchExceptions, checkPath}
 
-    Seq(
-      ("/", handlePublic(typesMap)),
+    val routes = Seq(
+      ("/assets/", handlePublic(typesMap)),
       ("/health", handleRoute(queryClient, "/health")),
       ("/api/v1/dependencies", handleRoute(queryClient, "/api/v1/dependencies")),
       ("/api/v1/services", handleRoute(queryClient, "/api/v1/services")),
       ("/api/v1/spans", handleRoute(queryClient, "/api/v1/spans")),
-      ("/api/v1/trace/:id", handleTrace(queryClient)),
+      ("/api/v1/trace/", handleTrace(queryClient)),
       ("/api/v1/traces", handleRoute(queryClient, "/api/v1/traces")),
       ("/config.json", handleConfig(Map(
         "environment" -> environment(),
         "queryLimit" -> queryLimit()
       )))
-    ).foldLeft(new HttpMuxer) { case (m , (p, handler)) =>
-      val path = p.split("/").toList
-      val handlePath = path.takeWhile { t => !(t.startsWith(":") || t.startsWith("?:")) }
-      val suffix = if (p.endsWith("/") || p.contains(":")) "/" else ""
+    )
 
-      m.withHandler(handlePath.mkString("/") + suffix,
-        collectStats(handlePath.foldLeft(stats) { case (s, p) => s.scope(p) }) andThen
+    val indexHtmlRoutes = (new ObjectMapper() with ScalaObjectMapper)
+      .registerModule(DefaultScalaModule)
+      .readValue(getClass.getResource("/zipkin-ui/index-routes.json").openStream(), classOf[Seq[String]])
```

Comment:
Since we're sunsetting zipkin-web, I suggest that we hard-code routes for serving index.html in here?
We could do the same with static assets.
Finatra, which is used in zipkin-query, solves this quite elegantly.

Another alternative is to rewrite zipkin-web to use the http proxy from here: https://twitter.github.io/finagle/guide/Quickstart.html
Since assets are served from zipkin-query, then zipkin-web could forward _all_ HTTP traffic to zipkin-query, as-is, without any route logic.